emhttp/plugins/dynamix.my.servers/include/state.php

```diff
@@ -83,12 +91,20 @@ public function __construct()
          * @see - getWebguiGlobal() for usage
          * */
         global $webguiGlobals;
-        $this->webguiGlobals =& $webguiGlobals;
+        $this->webguiGlobals = &$webguiGlobals;
         // echo "<pre>" . json_encode($this->webguiGlobals, JSON_PRETTY_PRINT) . "</pre>";
 
-        $this->var = (array)parse_ini_file('state/var.ini');
+        $this->var = $webguiGlobals['var'];
+
+        // If we're on a patch, we need to use the combinedVersion to check for updates
```

Comment:
Old logic flagging this line to make sure this gets updated.

Reply:
[`14e0fed` (#2099)](https://github.com/unraid/webgui/pull/2099/commits/14e0fed12a6cbdda0e5cd9c9cf7d37015085820e)